examples/ssh/windows_check_vpn_ext_hosts.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Domotz Custom Driver 
+ * Name: Windows check VPN connection
+ * Description: Check if a list of hosts are reachable from the endpoint where the script is run 
+ *   
+ * Communication protocol is SSH. Utilizing the native windows powershell command.
+ * 
+ * Tested on Windows Versions:
+ *  - Windows 10
+ *  - Microsoft Windows Server 2019
+ * 
+ * Powershell Version:
+ *  - 5.1.19041.2364
+ * 
+ * 
+ * Creates a Custom Driver Table with the following columns:
+ *  - Host
+ *  - Reachable
+ * 
+ *  Reachable can be [Yes, No]
+ * 
+**/
+
+// Define here the list of hosts/ips you would like to check
+var hostsToCheck = "www.notworkingnetaddress.com,www.google.com,www.microsoft.com.it,129.312.32.31" 
+
+// SSh command run on the endpoint
+var command="powershell -command \"Test-Connection -Cn "+hostsToCheck+" -BufferSize 16 -Count 1 -ea 0 -quiet \""
+
+// SSH options when running the commands
+var sshConfig = {
+    username: D.device.username(),
+    password: D.device.password(),
+    port: 22,
+    timeout: 10000
+};
+
+// SSH promise definition
+function checkSshError(err) {
+    if(err.message) console.error(err.message);
+    if(err.code == 5) D.failure(D.errorType.AUTHENTICATION_ERROR);
+    if(err.code == 255) D.failure(D.errorType.RESOURCE_UNAVAILABLE);
+    console.error(err);
+    D.failure(D.errorType.GENERIC_ERROR);
+}
+
+function executeCommand(command){
```

Comment:
Why define a helper function that you only use once?

You can just use the normal sshSend command with its respective callback

Reply:
wanted to preserve the standard skel we use to develop drivers on